src/Tribe/Views/V2/View.php

```diff
@@ -2194,6 +2194,12 @@ protected function should_show_recent_past_events_view() {
 	 */
 	protected function setup_additional_views( array $events = [], array $template_vars = [] ) {
 
+		// Only return if on the current or upcoming view and not in the past.
+		// todo this still displays on day view in the past if it is
```

Comment:
:no_entry_sign: Inline comments must end in full-stops, exclamation marks, or question marks

<sub>hash: 73a36701fca068a2393dbe337e79ad9e</sub>